ci/jenkins/test.sh

```diff
@@ -42,6 +42,8 @@ NETWORKPOLICY_SKIP="should allow egress access to server in CIDR block|should en
 
 CONTROL_PLANE_NODE_ROLE="master|control-plane"
 
+CLEAN_ANTREA_IMAGES_CONTAINERD="ctr -n k8s.io images list | grep 'antrea' | awk '{print $1}' | xargs -r ctr -n k8s.io images delete"
+CLEAN_DANGLING_IMAGES_CONTAINERD="ctr -n k8s.io images list | awk '{print $1}' | grep '^sha256:' | xargs -r ctr -n k8s.io images delete"
```

Comment:
and doesn't `CLEAN_STALE_IMAGES_CONTAINERD` already clean up unused image?

Reply:
Yes, CLEAN_STALE_IMAGES_CONTAINERD is enough for containerd node, updated